scripts/bundleSize.sh

```diff
@@ -2,8 +2,8 @@
 
 # Size limit for all bundles used by each service (K)
 # Keep these +/- 5K and update frequently!
-min=604
-max=622
+min=613
+max=631
```

Comment:
This PR increases the bundle size by around 9KB because of the bump to `@bbc/psammead-styles`

It was previously [`@0.3.0`](https://packagephobia.now.sh/result?p=@bbc/psammead-styles@0.3.0) which had an install size of 27.6KB and this bumps it to [`@1.1.2`](https://packagephobia.now.sh/result?p=@bbc/psammead-styles@1.1.2) which has an install size os 51.2KB. Once gzipped that roughly explains the 9KB difference

Reply:
![image](https://user-images.githubusercontent.com/11341355/60982711-92503b00-a330-11e9-842c-f081f340af3e.png)
Why?!

Reply:
This can also be checked in the `webpackBundleReport` as per the following screenshots:

On this branch 

![image](https://user-images.githubusercontent.com/7791726/60983918-ba409e00-a332-11e9-8642-674de016cfe1.png)


On latest

![image](https://user-images.githubusercontent.com/7791726/60984293-62566700-a333-11e9-8c6d-e7d3dc081317.png)

Reply:
Also there are multiple versions of `psammead-styles` which would explain why not all the 9KB is accounted for in the screenshots above